Problem2/largest_two_numbers.rb

```diff
@@ -0,0 +1,29 @@
+# File: largest_two_numbers.rb
+
+class LargestTwoNumbers
```

Comment:
`numbers.length < 2 ? (raise "Array of length too small, min 2!") : numbers.uniq.max(2)`

I think this should do it